src/plugins/data/server/ui_settings.ts

```diff
@@ -533,7 +533,7 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       value: `{
   "pause": false,
-  "value": 0
+  "value": 60
```

Comment:
![image](https://user-images.githubusercontent.com/3016806/121885103-6c9dee80-cd1c-11eb-8bf8-6ea150d12ff1.png)

I suspect you wanted 60 seconds, not 0.06 seconds ;-)